tools/eth1exporter/main.go

```diff
@@ -0,0 +1,173 @@
+// Prometheus exporter for Ethereum address balances.
+// Forked from https://github.com/hunterlong/ethexporter
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"math/big"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+var (
+	allWatching []*Watching
+	loadSeconds float64
+	totalLoaded int64
+	eth         *ethclient.Client
+)
+
+var (
+	port            = flag.Int("port", 9090, "Port to serve /metrics")
+	web3URL         = flag.String("web3-url", "https://goerli.prylabs.net", "Web3 URL to access information about ETH1")
```

Comment:
can we be consistent with this sort of flag? elsewhere in the repo we call it web3-provider. Just easier for sanity for anyone running this to have familiar flags.

Reply:
sg